flight_backend/api/tests/test_user_login_api.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+from rest_framework.test import APIClient, APITestCase
+from django.urls import reverse
+from rest_framework.views import status
+
+from api.models import User
+
+client = APIClient()
+
+
+class UserLogInTest(APITestCase):
+    def setUp(self):
+        self.url = reverse("log-in")
+        self.normal_user = User.objects.create_user(
+            first_name="Ghost",
+            last_name="Ghost",
+            email="ghost@gmail.com",
+            password="password1*",
+        )
+        self.login_details = {"email": "ghost@gmail.com", "password": "password1*"}
+
+    def test_successful_login(self):
+        response = client.post(
+            self.url,
+            data=json.dumps(self.login_details),
+            content_type="application/json",
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data["email"], "ghost@gmail.com")
```

Comment:
i think it'd be better to confirm the token is returned since this is what will be used in subsequent requests.

Reply:
Okay, Thanks. I am on it.